test/test_generated_site.rb

```diff
@@ -49,7 +49,9 @@ class TestGeneratedSite < JekyllUnitTest
     end
 
     should "include a post with a abbreviated dates" do
-      refute_nil -1, @site.posts.index { |post| post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md" }
+      refute_nil @site.posts.index { |post|
+        post.relative_path == "_posts/2017-2-5-i-dont-like-zeroes.md"
+      }
```

Comment:
In this case, I like to imaging parentheses around the argument, like `refute_nil(@site.posts.index...)`. The use of the curly braces indicate to me that this is used as an argument to `refute_nil`, whereas your example above doesn't read that directly. By diverging from the standard `do...end`, I call to attention that this statement is part of an argument.